src/driver/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["The Rust Project Developers"]
+name = "driver"
+version = "0.0.0"
```

Comment:
I'd prefer to for not not duplicate version numbers in too many locations, so I think it's fine to stick with 0.0.0. Going with Cargo as a build system is going to be experimental for awhile anyway.